src/core/strategy/mod.rs

```diff
@@ -0,0 +1,131 @@
+use crate::core::{ValueType, IndicatorResult, Action, Error, OHLCV};
```

Comment:
Contains all strategy related traits. Mostly copied from indicator traits. Can be optimized